Update exporting.md (#1659)
platformsh/platformsh-docs: docs/src/tutorials/exporting.md

The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.

To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.

> ## Get the environment variables

Reviewer: ```suggestion

## Get the environment variables
```

The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.

To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.

> 
> 

Reviewer: ```suggestion

```

Single line between paragraphs, not multiple.

## Get the environment variables

If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.

> As stated in the management console, several possibilities exist for the environment variables.
> • Variables beginning with `env`: will be exposed as Unix environment variables
> • Variables beginning with `php`: will be interpreted as php.ini directives.
> • All other variables will be part of the environment $PLATFORM_VARIABLES variable
> More details can be found [here](https://docs.platform.sh/administration/web/configure-environment.html#variables
> )

Reviewer: ```suggestion
As stated in the management console, several possibilities exist for the environment variables.

* Variables beginning with `env:` will be exposed as Unix environment variables
* Variables beginning with `php:` will be interpreted as `php.ini` directives.
* All other variables will be part of the environment `PLATFORM_VARIABLES` variable

More details can be found [on the environment page](https://docs.platform.sh/administration/web/configure-environment.html#variables)
```

The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.

To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.

## Get the environment variables

If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.

> As precised in the management console, several possibilities exist for the environment variables.

Reviewer: ```suggestion
As stated in the management console, several possibilities exist for the environment variables.
```

I think that's what you mean here?

Author: yes, my bad. "Stated" is much better.

> • Variables beginning with env: will be exposed as Unix environment variables

> • Variables beginning with php: will be interpreted as php.ini directives.

> • All other variables will be part of the environment $PLATFORM_VARIABLES variable
More details can be found [here](https://docs.platform.sh/administration/web/configure-environment.html#variables

> )

Reviewer: These should be a normal bulleted list, not a quoted pseudo-list.
